polaris-pipeline/coordinator/Functions/Orchestration/Client/Case/CaseClient.cs

```diff
@@ -111,83 +78,24 @@ public class CaseClient
-                        return orchestrationClient.CreateCheckStatusResponse(req, instanceId);
+                        return await _orchestrationProvider.RefreshCaseAsync(orchestrationClient, currentCorrelationId, caseId, casePayload, req);
 
                     case "DELETE":
-                        var telemetryEvent = new DeletedCaseEvent(
-                                correlationId: currentCorrelationId,
-                                caseId: caseIdNum,
-                                startTime: DateTime.UtcNow
-                        );
-                        try
-                        {
-                            await _searchIndexService.RemoveCaseIndexEntriesAsync(caseIdNum, currentCorrelationId);
-                            telemetryEvent.RemovedCaseIndexTime = DateTime.UtcNow;
-
-                            await _searchIndexService.WaitForCaseEmptyResultsAsync(caseIdNum, currentCorrelationId);
-                            telemetryEvent.IndexSettledTime = DateTime.UtcNow;
-
-                            // Terminate Orchestrations (can't terminate Durable Entities with Netherite backend, but can Purge - see below)
-                            var terminateOrchestrationQueries = GetOrchestrationQueries(terminateStatuses, caseId);
-                            var terminateOrchestrationInstanceIds = await TerminateOrchestrations(orchestrationClient, terminateOrchestrationQueries, currentCorrelationId);
-                            telemetryEvent.TerminatedInstancesCount = terminateOrchestrationInstanceIds.Count;
-                            telemetryEvent.GotTerminateInstancesTime = DateTime.UtcNow;
-                            telemetryEvent.TerminatedInstancesTime = DateTime.UtcNow;
-
-                            // Purge Orchestrations and Durable Entities
-                            var purgeConditions = GetOrchestrationQueries(purgeStatuses, caseId);
-                            purgeConditions.AddRange(GetDurableEntityQueries(terminateStatuses, caseId));
-                            var success = await Purge(orchestrationClient, purgeConditions, currentCorrelationId);
-                            telemetryEvent.EndTime = DateTime.UtcNow;
-
-                            _telemetryClient.TrackEvent(telemetryEvent);
-
-                            return new HttpResponseMessage(success ? HttpStatusCode.OK : HttpStatusCode.InternalServerError);
-                        }
-                        catch (Exception)
-                        {
-                            _telemetryClient.TrackEventFailure(telemetryEvent);
-                            throw;
-                        }
-
+                        return await _orchestrationProvider.DeleteCaseAsync(orchestrationClient, currentCorrelationId, caseIdNum);
 
                     case "PUT":
```

Comment:
Might be good to remove the PUT. It was used in testing to set the tracker, but that isn't possible now it's created from multiple source enties

Reply:
Done